net-im/purple-events/purple-events-0.99.2.ebuild

```diff
@@ -0,0 +1,24 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit meson
```

Comment:
Heh, the change to meson dropped SONAME for this package. I guess it's not intentional, but not sure whether it's breaking either. 

Could you try installing purple-events-0.99.1, then install x11-plugins/purple-libnotify-plus-2.99.1, upgrade to purple-events-0.99.2 and see if portage complains?

Reply:
A new SONAME is always breaking in that you have to do the subslot + := dance. You should speak to upstream if it wasn't intentional.

Reply:
Oh actually, upon closer inspection 0.99.1 installs `/usr/lib64/purple-2/` with two library files, while 0.99.2 does not. The only rdep doesn't seem to use these if it still compiles with 0.99.2 so I wonder how important this subdirectory is, and if it was ever supposed to exists.

Reply:
If they're plugins, it's no issue.